### DISCUSSION TITLE
Table name specified more than once

### DISCUSSION THREAD

**Discussion by DennisSchmidt2** [supabase/supabase#5954] [General]
Can someone please help me and explain how I can make this work:

```
final response = await Supabase.instance.client
        .from('transfers')
        .select('''
        id,
        playerUrl,
        firstName,
        lastName,
        club:from_club(id, name),
        fromClubUrl,
        club:to_club(id, name),
        toClubUrl,
        date,
        saison
        ''')
        .execute();
```

This is my query. The problem is with club:....

Is there vtl. a possibility to make nevertheless?

**Comment by GaryAustin1** [comment]
Edit:  "club" in that context is not table name, it is an alias for your link field name in the returned data.  Supabase/PostgREST figures out the table from the link field.

You have used "club" for an alias twice.  Try using just from_club and to_club without club: in front first, then alias each to something different if you want.
